clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -232,17 +245,33 @@ class AutoScalingWorker {
     }
 
     def autoScaling = regionScopedProvider.autoScaling
-    retrySupport.retry({ -> autoScaling.createAutoScalingGroup(request) }, 5, 1000, true)
+    Exception ex = retrySupport.retry({ ->
+      try {
+        autoScaling.createAutoScalingGroup(request)
+      } catch (AlreadyExistsException e) {
+        if (!shouldProceedWithExistingState(autoScaling, asgName, request)) {
+          return e
+        }
+        log.debug("Determined pre-existing ASG is desired state, continuing...", e)
+      }
+    }, 5, 1000, true)
```

Comment:
I normally don't do exponential backoff

Reply:
why not?

Reply:
I think I'll keep the exp backoff, but having a knob exposed inside of kork to adjust the exponential base like Asher suggests is a great idea.

Reply:
Seen here: https://github.com/spinnaker/kork/pull/141

Reply:
The AWS SDK is already retrying exponentially, I'm still not convinced that we need to layer our own exponential retries on top of that even with the ability to manipulate the base.

🤷‍♂️